src/controller/data_model/controller-clusters.matter

```diff
@@ -9134,6 +9134,56 @@ provisional cluster ContentAppObserver = 1296 {
   command ContentAppMessage(ContentAppMessageRequest): ContentAppMessageResponse = 0;
 }
 
+/** Supports the ability for clients to request the commissioning of themselves or other nodes onto a fabric which the cluster server can commission onto. */
+cluster CommissionerControl = 1873 {
+  revision 1; // NOTE: Default/not specifically set
+
+  bitmap SupportedDeviceCategoryBitmap : bitmap32 {
+    kFabricSynchronization = 0x1;
+  }
+
+  info event CommissioningRequestResult = 0 {
+    int64u requestId = 0;
+    node_id clientNodeId = 1;
+    enum8 statusCode = 2;
+  }
+
+  readonly attribute SupportedDeviceCategoryBitmap supportedDeviceCategories = 0;
```

Comment:
Access seems to be `R M` on this one, so I expect `access(write: manage)` here.

Also the commands are all manage access (instead of operate) in my copy of the spec. Was the XML manually written or using Alchecmy?

Reply:
It is manually written

Reply:
Alchecmy interpreted  access 'R M' as readonly
 
 readonly attribute access(read: manage) SupportedDeviceCategoryBitmap supportedDeviceCategories = 0;